e2e/go.mod

```diff
@@ -0,0 +1,142 @@
+module e2e
```

Comment:
Is this common practice? I would think the import should be "github.com/cosmos/ibc-go/e2e"? What would the differences be between using the different module paths?

Reply:
I was initially thinking of to do something like this, but opted to remove the `ibc-go` prefix to make it clear it is a separate entity. If you think it's better to have the fully qualified name we can always rename it.